site/docs/education/tutorials/postman-tutorial/overview.md

```diff
@@ -73,8 +126,8 @@ turn off certificate validation in postman:
 * Go to settings
 * Ensure 'SSL certificate verification' is switched off
```

Comment:
again I would associate the text and screenshot in a section

Reply:
I've given more detailed instructions, and included the screenshot in a collapsible section.